python/acronym/acronym.py

```diff
@@ -0,0 +1,13 @@
+def abbreviate(words):
+    words = words.upper()
+
+    crap = ',_-'
+
+    for c in crap:
+        words = words.replace(c, ' ')
+
+    acronym = ''
+
+    for w in words.split():
```

Comment:
This looks like our classic
```python
lst = []
for a in iter:
   if cond(a):
        b = f(a)
        lst.append(b)
```
which means... list comprehension!

Reply:
It's not in the code, not in this task. Did you mean a different one?

Reply:
```python
acronym = ''
for w in words.split():
    acronym += w[0]
return acronym
```
could you write that using a list comprehension and perhaps `.join()`?